code/datums/extensions/health/atom.dm

```diff
@@ -0,0 +1,411 @@
+/atom
+	/**
+	 * Whether or not the atom should use the health handler when initialized.
+	 */
+	var/use_health_handler = FALSE
+
+	/**
+	 * Initialization config for the health handler.
+	 *
+	 * `max_health` - Default maximum health.
+	 *
+	 * `resist_*` - Resistance multiplier for various damage types, where `*` is replaced by the damage type's string value.
+	 */
+	var/list/health_handler_config = list(
```

Comment:
No `= list()` vars on low types.

Reply:
Replaced with an overridable proc that returns a list.